src/node/coins_view_args.h

```diff
@@ -5,10 +5,25 @@
 #ifndef BITCOIN_NODE_COINS_VIEW_ARGS_H
 #define BITCOIN_NODE_COINS_VIEW_ARGS_H
 
+#include <cassert>
+#include <kernel/caches.h>
+
 class ArgsManager;
 struct CoinsViewOptions;
 
-namespace node {
+namespace node
+{
+static constexpr size_t GetDbBatchSize(const size_t dbcache_bytes)
+{
+    const auto [min, max]{std::pair{DEFAULT_DB_CACHE_BATCH, 256_MiB}};
+    assert(min < max); // std::clamp is undefined if `lo` is greater than `hi`
```

Comment:
Off by one

```suggestion
    assert(min <= max); // std::clamp is undefined if `lo` is greater than `hi`
```

Reply:
* the `lo` and `hi` refer to the clamp argument names
* if min == max the whole method could be eliminated